### ISSUE TITLE
How can I add a layer before a pre-trained model?

### ISSUE THREAD

**Issue by ssiaw12345** [fchollet/keras#3836]
I want to visualize the hidden layer of my pre-trained autoencoder like https://blog.keras.io/how-convolutional-neural-networks-see-the-world.html do, but I find that the amplitude of result will increase without stop, because there is no normalization for the input.
So, how can I add a BatchNormalization layer before my pre-trained autoencoder?

**Comment by keunwoochoi** [comment]
I haven't done the same, but I guess it should work by
* set layers names
* in `load_weights()` set the argument `by_name=True`.

**Comment by cwzat** [comment]
How you solve the problem?
